check_run/overrides/sales_invoice.py

```diff
@@ -0,0 +1,61 @@
+import frappe
+from erpnext.accounts.doctype.accounting_dimension.accounting_dimension import (
+	get_accounting_dimensions,
+)
+from erpnext.accounts.doctype.sales_invoice.sales_invoice import SalesInvoice
+from erpnext.accounts.party import get_due_date
+from erpnext.accounts.utils import get_account_currency
+from frappe.utils.data import cint, flt
+
+
+class CheckRunSalesInvoice(SalesInvoice):
+	# TODO: update due date in taxes with validate hook

```

Comment:
Here is a TODO that I do not know if you want to do now

Reply:
@fproldan I think this can be removed, but I will add a test to ensure it's working.